docs/attributes-registry/db.md

```diff
@@ -216,6 +217,19 @@ This group defines attributes for Azure Cosmos DB.
 | `Session` | session | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `Strong` | strong | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
+## Azure Cosmos DB Network level Attributes
+
+This group defines attributes for Azure Cosmos DB Network call.
+
+| Attribute | Type | Description | Examples | Stability |
+|---|---|---|---|---|
+| <a id="db-cosmosdb-network-host" href="#db-cosmosdb-network-host">`db.cosmosdb.network.host`</a> | string | Host from Gateway or Backend endpoint address | `rntbd://cdb-east-us.azure.document.com` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| <a id="db-cosmosdb-network-port" href="#db-cosmosdb-network-port">`db.cosmosdb.network.port`</a> | int | Port from Gateway or Backend endpoint address | `443` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| <a id="db-cosmosdb-network-region" href="#db-cosmosdb-network-region">`db.cosmosdb.network.region`</a> | string | Azure Cosmos DB Region where the network call was made | `WEST US 2` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Please add explanation whether it is the source or target region. Also not sure whether we would need both. For rntbd host when cx uses private endpoint the host would never indicate the target region.